docs/networks/software-versions.mdx

```diff
@@ -12,6 +12,42 @@ Release candidates are software releases that are also released to the [Testnet]
 
 [testnet]: ./README.mdx
 
+## Protocol 22 (Testnet, September 25, 2024)
+
+### Software
+
+| Software | Version |
+| --- | --- |
+| XDR | `v21.1` |
+| Rust XDR | `v21.0.1` |
+| Soroban Environment | `v21.0.2` |
+| Stellar Core | `v21.0.0` |
+| Soroban Rust SDK | `21.0.1-preview.3` |
+| Stellar CLI | `v21.0.0` |
+| Soroban RPC | `v21.3.0` |
+| Stellar Horizon | `v2.30.0` |
+| Stellar Quickstart | `https://hub.docker.com/layers/stellar/quickstart/v426-latest-amd64/images/sha256-274395daab6fa8033b9213f152d56699358917fb01d7c7e95392a37fc00c9d01` |
+| Stellar JS Stellar Base | `v12.0.0-rc1` |
+| Stellar JS Stellar SDK | `v12.1.0` |
+| Freighter |  |
+| Laboratory |  |
+| Soroban React Payment dapp | `TBD` |
```

Comment:
I wonder if we should remove these 3 dapp entries. They have been in TBD for quite some time.

Reply:
Works for me.